### PR TITLE
Updates to improve UI

### LINKED ISSUE #292: [PE-D][Tester E] Accessing view by time
It is not clear to me how one can access viewing by time

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: ollayf/ped#5

### LINKED ISSUE #294: [PE-D][Tester C] Documentation Bug: Ascending order
Userguide:

Code tryout:

The ascending order in the documentation is not clearly documented in detailed, is it ascending order based on the amount? Ascending order based on the date? Or ascending order based on the index? I'm unable to determine and understand it. 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: WilsonLee2000/ped#2

### LINKED ISSUE #307: [PE-D][Tester B] Monthly Budget doesn't propagate backwards or forwards
After setting the monthly budget, the budget does not carry forward to the next month.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: MingEn82/ped#6

### LINKED ISSUE #312: [PE-D][Tester D] Help guide in software slightly inconsistent
For example in `help shortcut`, it states the command to delete a shortcut is `<shortcut delete>`. However it is actually supposed to be using an underscore.

Also a slight typo in the help message for shortcut salary (misplaced space before comma)

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: choongzhanhong/ped#8

### LINKED ISSUE #325: [PE-D][Tester E] Negative budget issues
exceeded budget when negative

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: ollayf/ped#14

### LINKED ISSUE #332: [PE-D][Tester A] Incorrect capitalization in User Guide
The rest of your documentation uses `rainyDay`

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: ERJUNZE/ped#10

### LINKED ISSUE #334: [PE-D][Tester A] Incorrect order of dollar and negative symbol
The minus sign should be before the dollar sign for negative amounts, for example `-$7.00` instead of `$-7.00`

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: ERJUNZE/ped#11

### LINKED ISSUE #335: [PE-D][Tester C] Handle invalid Timespan
I tried to test the 0d as timespan and received the message "your financial report is empty" which is not really accurate, perhaps some code handling can be done to resolve such edge cases of 0d? 

-------------
Labels: `type.FeatureFlaw` `severity.Low`
original: WilsonLee2000/ped#7

### LINKED ISSUE #336: [PE-D][Tester A] Bye command not included in help
The list of commands within the application does not include the exit command, `bye`. Running `help bye` also does not produce a help message for using the command `bye`. Users will not know how to exit the application without referring to the user guide.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: ERJUNZE/ped#12

### PR DESCRIPTION
**Pull request by BenjaminPoh**
Fixes #336 : bye command added to help
Fixes #335 : timespans on view command now accurately implements inclusivity of start/end dates
Fixes #334 : negative symbol now behind $ in summary
Fixes #332 : RainyDay replaced with rainyDay
Fixes #325 : rainyDay now gives you an attitude if you change the budget via the datafile
Fixes #312 : updated help table for shortcut to improve clarity by stating the action in present continuous tense
Fixes #307 : Month/Year explicity stated regardless of the user's date now
Fixes #294 : Improve clarity of what ascending order means
Fixes #292 : UG includes examples for view command

### CODE REVIEW

File: src/main/java/seedu/rainyDay/command/HelpCommand.java [hunk 1]
```diff
@@ -37,8 +38,7 @@ public class HelpCommand extends Command {
             "| -c <CATEGORY> | Optional    | Used to denote the category of the entry. Set to Misc if omitted     |\n" +
             "| -date <DATE>  | Optional    | Used to denote the date of the entry. Set to present date if omitted |\n" +
             "+-----------------------------+----------------------------------------------------------------------+\n" +
-            "| Note: DESCRIPTION and CATEGORY cannot contain \"-\" character                                        |\n"
-            +
+            "| Note: DESCRIPTION and CATEGORY cannot contain \"-\" character                                      |\n" +
```
Comment (lil1n) [inline src/main/java/seedu/rainyDay/command/HelpCommand.java:41]: would the format for this row be shorter when shown to user? Due to the two additional `\` characters.

Comment (BenjaminPoh) [reply src/main/java/seedu/rainyDay/command/HelpCommand.java]: aaa you are right. Added comment as well as a reminder

**Review by lil1n — APPROVED** [review]
lgtm
